### ISSUE TITLE
Add New Work Form, Accessibility, and Screen Readers

### ISSUE THREAD

**Issue by hectorcorrea** [psu-libraries/scholarsphere-3#308]
The text "Agree to the deposit agreement and then select files.  Press the Start Upload Button once all files have been selected" was removed from the Upload form in Sufia 7. This text was previously marked with CSS class `sr-only` in Sufia 6. We need to add an equivalent helper for Screen Readers in Sufia 7.

**Comment by carolyncole** [comment]
The entire new work page needs a usability review.  This ticket is about checking the page using screen reader  and adding tickets.  Can not get to add files buttons via the tab has also been noted.

**Comment by carolyncole** [comment]
@mtribone has some browser plugins we could use for this

**Comment by mtribone** [comment]
Using the WAVE accessibility tool, we need to solve the 25 errors and 6 alerts as a first step. It's evaluating all of the tabs on the page.

![screen shot 2016-11-02 at 1 20 50 pm](https://cloud.githubusercontent.com/assets/4163828/19939403/39d62452-a0ff-11e6-8d81-457751ff3d03.png)

**Comment by mtribone** [comment]
Addressing some missing button values and exposing the help text for everyone and not just screen readers.

![screen shot 2016-11-03 at 1 05 10 pm](https://cloud.githubusercontent.com/assets/4163828/19976654/06b5fbf6-a1c7-11e6-85eb-bb0c8f38faf7.png)
